warehouse/azure-synapse/azure-synapse.go

```diff
@@ -611,6 +611,31 @@ func (as *HandleT) loadUserTables() (errorMap map[string]error) {
 	return
 }
 
+//Need to create a structure with delete parameters instead of simply adding a long list of params
+func (as *HandleT) DeleteBy(tableNames []string, jobRunID string, sourceID string, taskRunID string) (success bool, err error) {
```

Comment:
Why not only have a error as response and value of nil would be successful deletion right ?